backend/src/packages/mail/libs/views/views-mapper.ts

```diff
@@ -0,0 +1,8 @@
+import { TemplateName } from '../enums/enums.js';
+import { plainView } from './plain/plain.js';
+
+const ViewsMapper = {
```

Comment:
as it is a map, you should create libs/maps/maps, libs/maps/templateNameToView for example
and don't forget to add a type for this map